src/modules/form/components/viewable/DynamicViewField.tsx

```diff
@@ -103,9 +106,27 @@ const DynamicViewField: React.FC<DynamicViewFieldProps> = ({
         />
       );
     case ItemType.TimeOfDay:
+      return (
+        <TextContent
+          {...commonProps}
+          renderValue={(val) => formatTimeOfDay(val)}
+        />
+      );
     case ItemType.Text:
     case ItemType.String:
+      return <TextContent {...commonProps} />;
     case ItemType.Integer:
+      if (item.component === Component.MinutesDuration) {
+        return (
+          <TextContent
+            {...commonProps}
+            renderValue={(val) => {
+              const [hours, minutes] = minutesToHoursAndMinutes(val);
+              return formatDuration({ minutes, hours }, { zero: true });
```

Comment:
read-only display for minutes duration

<img width="208" alt="Screenshot 2024-02-22 at 3 49 19 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/66ac302c-150b-4fbb-9a7a-6442cf492697">